Grascale Example in Readme not working

[Grascale Example](https://github.com/kornia/kornia-rs/tree/main?tab=readme-ov-file#examples-image-processing) was not Updated. While running the example causing  multiple  compilation issues

> error[E0603]: enum `InterpolationMode` is private

> error[E0609]: no field `data` on type `kornia::kornia_image::Image<u8, 3>`

--- a/README.md
+++ b/README.md
@@ -139,12 +139,30 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     println!("gray_resize: {:?}", gray_resized.size());
 
     // create a Rerun recording stream
-    let rec = rerun::RecordingStreamBuilder::new("Kornia App").connect()?;
+    let rec = rerun::RecordingStreamBuilder::new("Kornia App").spawn()?;
 
-    // log the images
-    let _ = rec.log("image", &rerun::Image::try_from(image_viz.data)?);
-    let _ = rec.log("gray", &rerun::Image::try_from(gray.data)?);
-    let _ = rec.log("gray_resize", &rerun::Image::try_from(gray_resized.data)?);
+    rec.log(
+        "image",
+        &rerun::Image::from_elements(
+            image_viz.as_slice(),
+            image_viz.size().into(),
+            rerun::ColorModel::RGB,
+        ),
+    )?;
+
+    rec.log(
+        "gray",
+        &rerun::Image::from_elements(gray.as_slice(), gray.size().into(), rerun::ColorModel::L),
+    )?;
+
+    rec.log(
+        "gray_resize",
+        &rerun::Image::from_elements(
+            gray_resized.as_slice(),
+            gray_resized.size().into(),
+            rerun::ColorModel::L,
+        ),
+    )?;
 
     Ok(())
 }

--- a/README.md
+++ b/README.md
@@ -133,7 +133,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     let mut gray_resized = Image::<f32, 1>::from_size_val(new_size, 0.0)?;
     imgproc::resize::resize_native(
         &gray, &mut gray_resized,
-        imgproc::resize::InterpolationMode::Bilinear,
+        imgproc::interpolation::InterpolationMode::Bilinear,
     )?;
 
     println!("gray_resize: {:?}", gray_resized.size());

